src/pages/security/static/_scope_content.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import styled from 'styled-components'
+import device from 'themes/device'
+import { localize, Localize } from 'components/localization'
+import { Header, LocalizedLinkText } from 'components/elements'
+
+const LinkText = styled(LocalizedLinkText)`
+    font-size: 14px;
+
+    @media ${device.tabletL} {
+        font-size: 12px;
+    }
+`
+
+export const scope_content = [
+    {
+        title: localize('Important businesses'),
+        list_icon: 'tick',
+        description: '',
+        check_list: [
+            { content: localize('Our payment site: cashier.deriv.com') },
+            { content: localize('Our login site: oauth.deriv.com') },
+            { content: localize('Our WebSockets API: *.binaryws.com api.deriv.com') },
+            {
+                content: (
+                    <Localize
+                        translate_text="Our main trading platform: app.deriv.com<3>*</3>"
+                        components={[
+                            <Header key={3} type="paragraph-2" as="span" color="red" size="14px" />,
+                        ]}
+                    />
+                ),
+                description: (
+                    <Localize
+                        translate_text="<0>*</0>This only covers the functionalities handled by Deriv"
+                        components={[
+                            <Header as="span" color="red" type="small" weight="normal" key={0} />,
+                        ]}
+                    />
+                ),
+            },
+            { content: localize('Our legacy trading platform: smarttrader.deriv.com') },
+        ],
+    },
+    {
+        title: localize('General businesses'),
+        list_icon: 'tick',
+        description: '',
+        check_list: [
+            {
+                content: (
+                    <Localize
+                        translate_text="Our GitHub repositories: <0>github.com/binary-com</0>"
+                        components={[<Header key={0} type="paragraph-2" weight="normal" as="p" />]}
+                    />
+                ),
+            },
+            { content: localize('Our CFD trading application by Devexperts: dx.deriv.com') },
+            {
+                content: (
+                    <Localize
+                        translate_text="Deriv P2P: Our peer-to-peer payments app (<0>Android app</0>, <1>iOS app</1>)"
+                        components={[
+                            <LinkText
+                                key={0}
+                                to={'https://play.google.com/store/apps/details?id=com.deriv.dp2p'}
+                                target="_blank"
+                                color="blue-9"
```

Comment:
Please add the `external` prop for all external links. If you don't add it, it will render out `Link` component from Gatsby, which is meant for internal links. External links should be rendered with an `a` tag instead.